### PR TITLE
Feat : display random quotes & user onboard

### PR DESCRIPTION
**Pull request by yogesh-**
input
inside

### CODE REVIEW

**Review by syed-shaida-hussain — APPROVED** [review]
No big changes required except for removing console logs and using good naming convention.

File: src/components/greet.js [hunk 1]
```diff
@@ -9,17 +10,20 @@ export const Greet = () => {
       hour12: false,
     });
     console.log(windowTime);
```
Comment (nareshpoonia) [inline src/components/greet.js]: Console.log can be removed

File: src/components/onboarding.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+export const UserOnboard = () => {
+  const [name, setName] = useState("");
+
+  const saveName = (e) => {
+    if (e.key === "Enter" || name !== "") {
+      localStorage.setItem("username", name);
+    }
+  };
+  console.log("name", name);
```
Comment (nareshpoonia) [inline src/components/onboarding.js]: console.log can be removed

**Review by nareshpoonia — APPROVED** [review]
The code look clean, good to merge.
Console.log at couple of places can be removed

File: src/components/onboarding.js (hunk 2 above)
Comment (yogesh-) [reply src/components/onboarding.js]: ok

File: src/components/greet.js (hunk 1 above)
Comment (yogesh-) [reply src/components/greet.js]: ok

**Pull request merged**